api_tests/nodes/views/test_node_linked_registrations.py

```diff
@@ -7,421 +7,463 @@
+            return app.get(url, expect_errors=expect_errors)
+        return request
+
+    def test_can_vew_linked_registrations_relationship(self, make_request, registration, user_admin_contrib, user_write_contrib, user_read_contrib, user_non_contrib, node_public, node_private):
+
+    #   test_public_node_unauthenticated_user_can_view_linked_registrations_relationship
+        res = make_request(node_id=node_public._id)
+        assert res.status_code == 200
+        assert res.json['data'][0]['id'] == registration._id
+
+    #   test_private_node_admin_contributor_can_view_linked_registrations_relationship
+        res = make_request(node_id=node_private._id, auth=user_admin_contrib.auth)
+        assert res.status_code == 200
+        assert res.json['data'][0]['id'] == registration._id
+
+    #   test_private_node_rw_contributor_can_view_linked_registrations_relationship
+        res = make_request(node_id=node_private._id, auth=user_write_contrib.auth)
+        assert res.status_code == 200
+        assert res.json['data'][0]['id'] == registration._id
+
+    #   test_private_node_read_contributor_can_view_linked_registrations_relationship
+        res = make_request(node_id=node_private._id, auth=user_read_contrib.auth)
+        assert res.status_code == 200
+        assert res.json['data'][0]['id'] == registration._id
+
+    #   test_private_node_non_contributor_cannot_view_linked_registrations_relationship
+        res = make_request(node_id=node_private._id, auth=user_non_contrib.auth, expect_errors=True)
+        assert res.status_code == 403
+        assert res.json['errors'][0]['detail'] == exceptions.PermissionDenied.default_detail
+
+    #   test_private_node_unauthenticated_user_cannot_view_linked_registrations_relationship
+        res = make_request(node_id=node_private._id, expect_errors=True)
+        assert res.status_code == 401
+        assert res.json['errors'][0]['detail'] == exceptions.NotAuthenticated.default_detail
+
+
+@pytest.mark.django_db
 class TestNodeLinkedRegistrationsRelationshipCreate(LinkedRegistrationsTestCase):
 
-    def setUp(self):
-        super(TestNodeLinkedRegistrationsRelationshipCreate, self).setUp()
-
-    def make_payload(self, registration_id=None):
-        return {
-            'data': [{
-                'type': 'linked_registrations',
-                'id': registration_id
-            }]
-        }
-
-    def make_request(self, node_id=None, auth=None, reg_id=None, expect_errors=False):
-        url = '/{}nodes/{}/relationships/linked_registrations/'.format(API_BASE, node_id)
-        if auth:
-            return self.app.post_json_api(url, self.make_payload(registration_id=reg_id), auth=auth, expect_errors=expect_errors)
-        return self.app.post_json_api(url, self.make_payload(registration_id=reg_id), expect_errors=expect_errors)
-
-    def test_admin_contributor_can_create_linked_registrations_relationship(self):
+    @pytest.fixture()
```

Comment:
Not sure why these need to be fixtures, does it save any time? Looks like it could just be a payload function not wrapped, though no strong feelings if you decide not to change.